ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -173,3 +175,10 @@ Verify Pipeline Run Deployment Is Successful    # robocop: disable
     ${containerStatuses}=  Create List        terminated    terminated
     ...    terminated    terminated    terminated
     Verify Deployment    ${valid_model}  1  1  ${containerNames}    ${podStatuses}    ${containerStatuses}
+
+Verify Data Science Parameter From A Duplicated Run Are The Same From The Compiled File
+    [Documentation]    Verify Data Science Parameter From A Duplicated Run Are The Same From The Compiled File
+    ${input_parameters}=    Open Pipeline Run Duplicate Page    ${PIPELINE_TEST_RUN_BASENAME}
```

Comment:
sounds this kw should be more named sth like "get input parameters" rather then `Open ..`